### ISSUE TITLE
Software support for setup? 

### ISSUE THREAD

**Issue by Wishingtreetops** [Mammoth-3D/Mammoth-FDM-MODs#1]
Where can I go for support for setting the software side up? I've followed the vez3D YouTube video on it. But it's only powering up, I can't actually control it.. but I followed his instructions. Only thing I did differently was use a fan slot on my BTT board, and HE1 slot for powering it (I didn't use the rpi as an MCU like he did in the video)

**Comment by Wishingtreetops** [comment]
Here is my thread I've posted a few places to no avail.
Okay so I am having a couple issues with my CPAP fan ive setup.. maybe someone who knows a little more about the core functionalities of these units can help? I still have my normal partcooling fan hooked up, right now im just testing the CPAP before mounting.... so I hooked up my control wire to my PD13 which is already defined in my MCU definitions as EXHAUST_FAN, I then hooked up my power wires to PA3 which is the HE1 slot, which isnt defined in my MCU.. but I still set it up. . . . I added to my cfg this "[fan_generic EXHAUST_FAN]
pin: PD13
max_power: 1
cycle_time: 0.002
shutdown_speed: 0"  and then I added                                                                        [output_pin CPAPpowerfeed]
pin: PA3
Value: 1
Shutdown_value:0 .... So When i power up, it turns on right away full blast, and now i see a exhaust_fan slider on my fan selection area in mainsail, but its already at 0 and sliding it doesnt do anything for control? not sure whats missing , hope someone here can direct me... its gotta be something not correct with my fan config above, since it powers on but it's not controllable?

Printer voron trident
Cooling fan CPAP WM7040
MCU board is BTT octopus 1.1
Klipper installed on raspie3b+
